index.js

```diff
@@ -1,19 +1,39 @@
+const yaml = require('js-yaml');
+
 module.exports = robot => {
     robot.on('pull_request.opened', receive);
     robot.on('issues.opened', receive);
     async function receive(context) {
-        const title = context.payload.issue.title;
-        const body = context.payload.issue.body;
-        // robot.log('Title: ', title, 'Body: ', body);
-
-        // Add options for if there is a default title?
+        let title, body, number, bodies, badTitle;
```

Comment:
Pardon the review pre ☕️  but what's the difference between `body` and `bodies`?